zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -431,6 +432,28 @@ public void close() {
     }
   }
 
+  public Boolean isDefaultRepoGit() {
```

Comment:
Could be {return isRepoGit(0)}. And in isRepoGit maybe a better error message for catching of IOException.

Reply:
Yes, that sounds simpler, thanks.